redhat-operators/codeready-workspaces/codeready-workspaces.1.1.0.csv.yaml

```diff
@@ -0,0 +1,370 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
@eivantsov Since the file is named as *.csv.yaml , we might need this https://github.com/operator-framework/operatorhub.io/pull/97

Reply:
Or you would have to change the name to *.clusterserviceversion.yaml

Reply:
@sbose78 I did not know about this naming convention. I basically looked at examples earlier *(maybe a few months ago) and saw *.csv. Anyway, Ienamed the file. Thanks.

Btw, RH operators do not show up in operatohub.io. At least there's no AMQ  https://operatorhub.io/?keyword=AMQ

Reply:
Good catch @sbose78 ! Can you change the filename for now? I will work on supporting any filename (by just inspecting the file for the kind) soon and keep you informed

Reply:
Actually please disregard. @eivantsov is right, RH operators don't show up in operatorhub.io

Reply:
I will still work on supporting any filename soon

Reply:
Ah Ok, RH  operators go into the Quay App Registry only?

Reply:
Yeah